corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -221,34 +221,34 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
                 </div>
               </div>
             </div>
-            {% if request|toggle_enabled:"ROLE_WEBAPPS_PERMISSIONS" %}
+            {% if request|toggle_enabled:"ROLE_APP_ACCESS_PERMISSIONS" %}
               <div class="form-group">
                 <label class="col-sm-4 control-label">
-                  {% trans "View All Web Apps" %}
+                  {% trans "Access All Applications" %}
```

Comment:
How about updating the name or at least the description to make it clearer that "access" means "data entry" and no access in HQ? Maybe "Data Entry for All Applications" and then have the description explain that users can enter data both via mobile devices and web apps?

Reply:
Although we are blocking restores if this FF is enabled but as per the user's perspective we actually restricting users from even logging in, that's why I have used "access". Should we update it to "data entry"?